source/common/safe_init/target_impl.cc

```diff
@@ -0,0 +1,56 @@
+#include "common/safe_init/target_impl.h"
+
+namespace Envoy {
+namespace SafeInit {
+
+using std::placeholders::_1;
```

Comment:
Is this used? FWIW, in general, I find `std::bind` to be confusing and from a time before there was built-in lambda support in the language. I don't feel that strongly about it, but I would probably replace the use of std::bind below with a lambda.

Reply:
Oh, good catch. I'd used it in an earlier iteration and forgot to remove this `using` declaration. Will clean up.